common/styles/appfooter.css

```diff
@@ -0,0 +1,12 @@
+footer .container1 {
```

Comment:
Is there a reason that we have three different css files? `app.css`, `appheader.css` and `appfooter.css`? The only reason that I can see for having header in separate file is that search app is using "common/appheader.css" but not the "common/app.css". But it seems like we are not showing this new footer in search app.